Modelica/package.mo

```diff
@@ -7937,8 +7937,8 @@ class Version_1_5 "Version 1.5 (Dec. 16, 2002)"
 <ul>
 <li>All example models made <strong>encapsulated</strong></li>
 <li>Upper case constants changed to lower case (cf. Modelica.Constants)</li>
-<li>Introduced Modelica.SIunits.Wavelength due to typo \"Wavelenght\"</li>
-<li>Introduced ModelicaAdditions.Blocks.Logical.Interfaces.Comparison due to typo \"Comparision\"</li>
+<li>Introduced Modelica.SIunits.Wavelength due to typo \"Wavelength\"</li>
+<li>Introduced ModelicaAdditions.Blocks.Logical.Interfaces.Comparison due to typo \"Comparison\"</li>
```

Comment:
Here the old typo is explicitly mentioned. I propose to remove them, as otherwise they are detected again and again.

Reply:
Makes sense. Will push right away.